code/_globalvars/lists/admin.dm

```diff
@@ -0,0 +1,10 @@
+/proc/init_smites()
+	var/list/smites = list()
+	for (var/_smite_path in subtypesof(/datum/smite))
+		var/datum/smite/smite_path = _smite_path
+		smites[initial(smite_path.name)] = smite_path
+	return smites
+
+GLOBAL_LIST_INIT_TYPED(smites, /datum/smite, init_smites())
```

Comment:
I'd like you to make it some less-common name for this glob. like `admin_smites`

Reply:
you want me to change the list of smites, to be named admin_smites?

There is no other GLOB list of smites, so I'm confused. Why would someone mix it up? 

Doing this just seems needlessly verbose

Reply:
yes I am talking literally like `GLOB.admin_smites`. 
I prefer slightly verbose variable names because it's better to identify when new variables having a similar name are added in the future rather than editing their name later.

Reply:
admin_smites would be a poor choice.

smites, init_smites, _smite_path, smite_path all share the same naming scheme as what they are referencing `datum/smite`. So I will rename it to `smite_list`